doc/source/reference/c-api/array.rst

```diff
@@ -3136,7 +3136,7 @@ the C-API is needed then some additional steps must be taken.
           be defined in another compilation unit.
         * Whenever :c:macro:`PY_ARRAY_UNIQUE_SYMBOL` is #defined, it
           also changes the name of the variable holding the C-API, which
-          defaults to :c:data:`PyArray_API`, to whatever the macro is
+          defaults to ``PyArray_API``, to whatever the macro is
```

Comment:
If anything else refers to this, it might be worth creating the link target rather than removong the reference

Reply:
So far, `PyArray_API` is not referred from other parts.